lib/rucio/daemons/conveyor/common.py

```diff
@@ -262,11 +265,15 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, source_strateg
                   'selection_strategy': source_strategy if source_strategy else activity_source_strategy.get(str(transfer['file_metadata']['activity']), default_source_strategy),
                   'request_type': transfer['file_metadata'].get('request_type', None),
                   'activity': str(transfer['file_metadata']['activity'])}
+
         if verify_checksum != 'none':
-            if 'md5' in list(t_file['metadata'].keys()) and t_file['metadata']['md5']:
-                t_file['checksum'] = 'MD5:%s' % str(t_file['metadata']['md5'])
-            if 'adler32' in list(t_file['metadata'].keys()) and t_file['metadata']['adler32']:
-                t_file['checksum'] = 'ADLER32:%s' % str(t_file['metadata']['adler32'])
+            rse_id = get_rse_id(transfer['rse'], session=session)
```

Comment:
Are you sure the `rse` key is being used here? (I didn't check, but usually this is dest_rse or something like that.

Reply:
I still have to test it, however I'll get access to a GridFTP and a SRM endpoints by the end of the week, so that I can perform some testing before giving your green light for merging 👍